TDP-3993: Audio BXML in conference callbacks (#442)
Bandwidth/bandwidth.github.io: voice/bxml/callbacks/conferenceCreated.md

{% method %}
## Conference Created Event
The Conference Created event is fired whenever a new [conference](../verbs/conference.md) that specified a callbackUrl is created.

> `<PlayAudio>` and `<SpeakSentence>` verbs can be used in BXML returned by this callback to play audio in the conference, which will be heard by members that enter the conference while still playing.

Reviewer: Let's make it clear that you can have no content at all, or you can have BXML. As it is, it sounds like you might have to return `<Response/>` if you want to do nothing when that isn't the case

Author: Reworded a bit, adding "can be either empty or a BXML" to the beginning.

### Expected response

```http
HTTP/1.1 200
Content-Type: application/xml; charset=utf-8

<Response>
    <!-- <PlayAudio> or <SpeakSentence> BXML verbs to play in the conference -->

> </Response>

Reviewer: Might be nice to have examples of no response and a response

Author: Added back the 204 response example.

{% method %}
## Conference Created Event
The Conference Created event is fired whenever a new [conference](../verbs/conference.md) that specified a `callbackUrl` is created.

> 

Reviewer 2: I don't think there should be a blank line here, though I do understand it might look weird with how the boxes around '<PlayAudio>' and '<SpeakSentence>' interact. So ignore this if that's the case

> 

Reviewer 2: I don't think there should be a blank line here, though I do understand it might look weird with how the boxes around PlayAudio and SpeakSentence clip into the line above it. So ignore this if that's the case

Reviewer 2: Same for the other callbacks as well.

Author: Huh, I just thought that the subject is different (when it's fired vs. accepted response) so made sense to separate.

> The response may be either empty or a BXML. Only `<PlayAudio>` and `<SpeakSentence>` are supported, which will be heard by members that enter the conference while still playing.

Reviewer 3: ```suggestion
The response may be either empty or a BXML. Only `<PlayAudio>` and `<SpeakSentence>` verbs are supported, which will be heard by all members of the conference.
```

Author: I wrote it that way because this is when the conference just started. I thought would be relevant to say that members who enter afterwards will hear it if it's still playing.
